docs/specs/tripgo.swagger.yaml

```diff
@@ -3293,6 +3293,17 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/NearbyCarParkLocation'
+      externalData:
+        type: object
+        description: It is a dictionary from "operatorName" and specific inforamtion from that operator.
+        properties:
+          $operatorName:
+            $ref: '#/definitions/ExternalData'
+        example:
+          ptv-realtime:
+            routeId: 2903,
```

Comment:
Looks weird to me with commas at the end of a YAML line

Reply:
Yes, I have removed them